### ISSUE TITLE
Feature request: "include_top=True/False" to allow customization of final layers

### ISSUE THREAD

**Issue by nwschurink** [shijianjian/EfficientNet-PyTorch-3D#3]
In the original tensorflow implementation an option called `include_top` can be passed as an argument to the model constructor.
When set to `False`, the final layers of the model are ommitted, which would allow easy implementation of one's own classification layers on top of the feature extractor.

My suggestion would be to add a single line here https://github.com/shijianjian/EfficientNet-PyTorch-3D/blob/master/efficientnet_pytorch_3d/model.py#L192 such that it becomes;

```
if self._global_params.include_top:
        # Pooling and final linear layer
        x = self._avg_pooling(x)
        x = x.view(bs, -1)
        x = self._dropout(x)
        x = self._fc(x)

return x
```
The `include_top` argument can default to `True` and then can be set to `False` using the `overide_params` option in the `from_name` function if one would like to just get the EfficientNet as a feature extractor.

This can be added easily by changing these line https://github.com/shijianjian/EfficientNet-PyTorch-3D/blob/master/efficientnet_pytorch_3d/utils.py#L20-L24 to

```
# Parameters for the entire model (stem, all blocks, and head)
GlobalParams = collections.namedtuple('GlobalParams', [
    'batch_norm_momentum', 'batch_norm_epsilon', 'dropout_rate',
    'num_classes', 'width_coefficient', 'depth_coefficient',
    'depth_divisor', 'min_depth', 'drop_connect_rate', 'image_size', 'include_top'])
```

and these lines https://github.com/shijianjian/EfficientNet-PyTorch-3D/blob/master/efficientnet_pytorch_3d/utils.py#L256-L282 to:

```
def efficientnet3d(width_coefficient=None, depth_coefficient=None, dropout_rate=0.2,
                 drop_connect_rate=0.2, image_size=None, num_classes=1000, include_top=True):
    """ Creates a efficientnet model. """

    blocks_args = [
        'r1_k3_s222_e1_i32_o16_se0.25', 'r2_k3_s222_e6_i16_o24_se0.25',
        'r2_k5_s222_e6_i24_o40_se0.25', 'r3_k3_s222_e6_i40_o80_se0.25',
        'r3_k5_s111_e6_i80_o112_se0.25', 'r4_k5_s222_e6_i112_o192_se0.25',
        'r1_k3_s111_e6_i192_o320_se0.25',
    ]
    blocks_args = BlockDecoder.decode(blocks_args)

    global_params = GlobalParams(
        batch_norm_momentum=0.99,
        batch_norm_epsilon=1e-3,
        dropout_rate=dropout_rate,
        drop_connect_rate=drop_connect_rate,
        # data_format='channels_last',  # removed, this is always true in PyTorch
        num_classes=num_classes,
        width_coefficient=width_coefficient,
        depth_coefficient=depth_coefficient,
        depth_divisor=8,
        min_depth=None,
        image_size=image_size,
        include_top=include_top
    )

    return blocks_args, global_params
```

**Comment by nwschurink** [comment]
Hi there,

I've added the functionality and have posted a pull request https://github.com/shijianjian/EfficientNet-PyTorch-3D/pull/4 if you are interested to merge this into your package.

**Comment by shijianjian** [comment]
Closed by #4.
